modules/splitter/monitoring-the-solver-proc.adoc

```diff
@@ -134,3 +134,46 @@ of the measuring.
 * `optaplanner.solver.solve-length.seconds-max`: run time of the
 longest-running currently active solver.
 * `optaplanner.solver.solve-length.seconds-duration-sum`: the sum of each active solver's solve duration. For example, if there are two active solvers, one running for three minutes and the other for one minute, the total solve time is four minutes.
+
+== Additional Metrics
+
+For more detailed monitoring, you can configure {PLANNER_SHORT} in the solver configuration to monitor additional metrics at a performance cost. The following example uses the `BEST_SCORE` and `SCORE_CALCULATION_COUNT` metric :
+
+[source,xml,options="nowrap"]
+----
+<solver xmlns="https://www.optaplanner.org/xsd/solver" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="https://www.optaplanner.org/xsd/solver https://www.optaplanner.org/xsd/solver/solver.xsd">
+  <monitoring>
+    <metric>BEST_SCORE</metric>
+    <metric>SCORE_CALCULATION_COUNT</metric>
+    ...
+  </monitoring>
+  ...
+</solver>
+----
+
+You can enable the following metrics in this configuration:
+
+* `SOLVE_DURATION` (enabled by default, Micrometer meter ID: `optaplanner.solver.solve.duration`): Measures the duration of solving for the longest active solver, the number of active solvers, and the cumulative duration of all active solvers.
+
+* `ERROR_COUNT` (enabled by default, Micrometer meter ID: `optaplanner.solver.errors`): Measures the number of errors that occurred while solving.
+
+* `SCORE_CALCULATION_COUNT` (enabled by default, Micrometer meter ID: `optaplanner.solver.score.calculation.count`): Measures the number of score calculations the {PLANNER_SHORT} performed.
+
+* `BEST_SCORE` (Micrometer meter ID: `optaplanner.solver.best.score.*`): Measures the score of the best solution that {PLANNER_SHORT} has found so far. There are separate meters for each level of the score. For instance, for a `HardSoftScore`, there are `optaplanner.solver.best.score.hard.score` and `optaplanner.solver.best.score.soft.score` meters.
+
+* `STEP_SCORE` (Micrometer meter ID: `optaplanner.solver.step.score.*`): Measures the score of each step that {PLANNER_SHORT} takes. There are separate meters for each level of the score. For instance, for a `HardSoftScore`, there are `optaplanner.solver.step.score.hard.score` and `optaplanner.solver.step.score.soft.score` meters.
+
+* `BEST_SOLUTION_MUTATION` (Micrometer meter ID: `optaplanner.solver.best.solution.mutation`): Measures the number of changed planning variables between consecutive best solutions.
+
+* `MOVE_COUNT_PER_STEP` (Micrometer meter ID: `optaplanner.solver.step.move.count`): Measures the number of moves evaluated in a step.
+
+* `MEMORY_USE` (Micrometer meter ID: `jvm.memory.used`): Measures the amount of memory used across the JVM. This does not measure the amount of memory used by a solver; two solvers on the same JVM will report the same value for this metric.
```

Comment:
Just caught another one - "This does not measure" is not in line with the IBM SG. Suggestion: "This metric does not measure"